website/package.json

```diff
@@ -13,7 +13,9 @@
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
     "fetch-popular-issues": "./scripts/fetch-popular-issues.sh",
-    "typecheck": "tsc"
+    "typecheck": "tsc",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "eslint . --ext .ts,.tsx --fix"
```

Comment:
Is it natural to use `:` as namespace (?) separator in `package.json`?

Reply:
I think this is a kind of common ways.
- https://github.com/honojs/hono/blob/main/package.json#L24
- https://github.com/facebook/docusaurus/blob/main/package.json#L63

On the other hand, in some projects, they are using `-` instead of `:`.
- https://github.com/facebook/react/blob/main/package.json#L131
- https://github.com/vercel/next.js/blob/canary/package.json#L51

Reply:
OK. Let's use `:`.